crescent/commands/decorators.py

```diff
@@ -98,6 +98,7 @@ def command(
                 continue
 
             generated = v._gen_option(n)
+            generated.name = generated.name.lower()
```

Comment:
I don't like this much. I think it would be better to do ```py
option_name = generated.name.lower()
```
and then just reference option_name.

Just FYI, `option_name` is just a random var idk what this actually is lol.

Reply:
Uh how does that work, we're just modifying the CommandOption from Hikari...

Reply:
Resolved based on feedback on Discord